src/feature/stats/rephist.c

```diff
@@ -1527,6 +1533,52 @@ rep_hist_load_state(or_state_t *state, char **err)
   return 0;
 }
 
+/** Collects BandwidthStatistics parameter from consensus.
+ *  Returns 0 iff BandwidthStatistics is 0 or
+ *  the consensus param is 0. */
+bool
+get_bandwidth_stats_param(void)
+{
+  int bandwidth_stats = get_options()->BandwidthStatistics;
+  switch (bandwidth_stats) {
+    case 0:
+    case 1:
+      return bandwidth_stats;
+  }
+  bandwidth_stats = networkstatus_get_param(NULL,
+                      "BandwidthStatistics", bandwidth_stats, 0, 1);
```

Comment:
We don't pass the value of the option as the consensus parameter default.

The function is supposed to return a number between 0 and 1 (min and max), so giving it -1 is weird.

Reply:
I think you might have missed this comment?